### PR TITLE
[Term Entry] UI and UX Design UIUX (Ideation) Journey Maps (Issue #3950)

### PR DESCRIPTION
**Pull request by DevGat718**
### Description

I'm working on journey mapping in ux & will be describing what the term means. 

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by DevGat718** [comment]
Good for now need some feedback. Id say i'm done.

**Comment by dakshdeepHERE** [comment]
@DevGat718 can you remove the other files that are pushed in this branch which is not needed here?

**Comment by DevGat718** [comment]
by files you mean the other commits? @dakshdeepHERE

**Comment by DevGat718** [comment]
I deleted the old branches

**Comment by dakshdeepHERE** [comment]
Yes exactly @DevGat718

**Review by dakshdeepHERE — COMMENTED** [review]
This file 
```
content/ai/concepts/foundation-models/terms/diffusion-models/diffusion-models.md
```
and this image still exist in this PR @DevGat718 
```
media/ai-diffusion-model-example.png
```

**Comment by DevGat718** [comment]
> This file
> 
> ```
> content/ai/concepts/foundation-models/terms/diffusion-models/diffusion-models.md
> ```
> 
> and this image still exist in this PR @DevGat718
> 
> ```
> media/ai-diffusion-model-example.png
> ```

so delete that file in git then commit it?

**Comment by dakshdeepHERE** [comment]
@DevGat718 yes correct. This issue is for UI UX right? Then remove those AI entries from this PR.

**Comment by DevGat718** [comment]
> @DevGat718 yes correct. This issue is for UI UX right? Then remove those AI entries from this PR.

yes this a ui/ux issue, but to be honest idk how those ai entries even got in here.

**Comment by dakshdeepHERE** [comment]
@DevGat718 just delete those file in this branch and push the changes.

**Comment by DevGat718** [comment]
done

File: content/uiux/journey-map/journey-map.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+Title: 'Journey Maps in UX'
+Description: ''
```
Comment (dakshdeepHERE) [inline content/uiux/journey-map/journey-map.md:3]: give small description here please

File: content/uiux/journey-map/journey-map.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'UI'
+- 'UX'
+- 'Design'
```
Comment (dakshdeepHERE) [inline content/uiux/journey-map/journey-map.md:10]: can you please explain the `tag.md` folder changes that you've done? Alot of those changes are not used here.

File: content/uiux/journey-map/journey-map.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+### Journey Maps
```
Comment (dakshdeepHERE) [inline content/uiux/journey-map/journey-map.md:16]:
```suggestion
```

File: content/uiux/journey-map/journey-map.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+A journey map is a visualization of the process that a person goes through in order to accomplish a goal. In basic form the journey mapping starts by compiling a series of user actions into a timeline. Then, the timeline is fleshed out with user thoughts and emotions in order to create a narrative which best fits that users persona throughout the customers journey. 
+
+**How They Are Used In UX**
```
Comment (dakshdeepHERE) [inline content/uiux/journey-map/journey-map.md:19]:
```suggestion
## How Journey map is used in UX

```

File: content/uiux/journey-map/journey-map.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+ The Journey Map, helps teams understand and visualize the end-to-end experience of a user as they interact with a product or service. They provide a comprehensive view of the user's interactions, emotions, and touch points throughout their journey.
```
Comment (dakshdeepHERE) [inline content/uiux/journey-map/journey-map.md:20]:
```suggestion
 The Journey Map, helps teams understand and visualize the end-to-end experience of a user as they interact with a product or service. They provide a comprehensive view of the user's interactions, emotions, and touchpoints throughout their journey.
 
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Please work on these few new changes requests @DevGat718 🚀

File: content/uiux/journey-map/journey-map.md (hunk 2 above)
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md:10]: I only added (ui ux and design) I figured those are appropraite tags for the doc I am working on. no?

File: content/uiux/journey-map/journey-map.md (hunk 1 above)
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md]: done

File: content/uiux/journey-map/journey-map.md (hunk 3 above)
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md:16]: done

File: content/uiux/journey-map/journey-map.md (hunk 4 above)
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md:19]: done

File: content/uiux/journey-map/journey-map.md (hunk 5 above)
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md]: done

**Comment by DevGat718** [comment]
ready for review, sorry I took so long been busy.

File: content/uiux/journey-map/journey-map.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A journey map is a visualization of the process that a person goes through in order to accomplish a goal. In basic form the journey mapping starts by compiling a series of user actions into a timeline. Then, the timeline is fleshed out with user thoughts and emotions in order to create a narrative which best fits that users persona throughout the customers journey. 
+
+## How Journey map is used in UX**
+ The Journey Map, helps teams understand and visualize the end-to-end experience of a user as they interact with a product or service. They provide a comprehensive view of the user's interactions, emotions, and touchpoints throughout their journey. These maps are a versatile tool in UX design that aids in understanding, visualizing, and improving the overall user experience by providing a holistic view of the user's journey.
+![Journey map](../../../media/journey_map#2.jpg)
```
Comment (dakshdeepHERE) [inline content/uiux/journey-map/journey-map.md:20]: The link should look like this instead of path. Also instead of that `#` use `-` in the image name.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
One last comment for you to work on

File: content/uiux/journey-map/journey-map.md (hunk 6 above)
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md]: done

**Comment by DevGat718** [comment]
lets get this published!

**Comment by dakshdeepHERE** [comment]
@DevGat718 first work on the comments. I told you correct the path link for the image and the name of the image. This is what a link should look like `https://raw.githubusercontent.com/Codecademy/docs/main/media/border-dotted-red-example.png`. You are using the path.

**Comment by DevGat718** [comment]
> @DevGat718 first work on the comments. I told you correct the path link for the image and the name of the image. This is what a link should look like `https://raw.githubusercontent.com/Codecademy/docs/main/media/border-dotted-red-example.png`. You are using the path.

which image link is that? & didn't see any comment talking about correct a path link. Only think about an image you told me was to delete the AI one & to put a - over the #

**Comment by dakshdeepHERE** [comment]
@DevGat718 leave the link I will do it from my end. You just rename the image file like `journey-map.jpg`.

**Comment by DevGat718** [comment]
done @dakshdeepHERE

**Comment by DevGat718** [comment]
we good? @dakshdeepHERE

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀

**Comment by DevGat718** [comment]
is it live yet?

**Comment by DevGat718** [comment]
can I get a update on this project, I would like too move on to the next. @ishg-153 @dakshdeepHERE

File: content/uiux/journey-map/journey-map.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A journey map is a visualization of a person's process to accomplish a goal. In basic form, the journey mapping starts by compiling a series of user actions into a timeline. Then, the timeline is fleshed out with user thoughts and emotions to create a narrative that best fits that user's persona throughout the customer's journey. 
+
+## How Journey map is used in UX**
+ The Journey Map, helps teams understand and visualize the end-to-end experience of a user as they interact with a product or service. They provide a comprehensive view of the user's interactions, emotions, and touchpoints throughout their journey. These maps are a versatile tool in UX design that aids in understanding, visualizing, and improving the overall user experience by providing a holistic view of the user's journey.
+![Journey map](https://raw.githubusercontent.com/Codecademy/docs/main/media/journey-map.jpg)
```
Comment (ishg-153) [inline content/uiux/journey-map/journey-map.md:20]:
```suggestion

![Steps to follow in creating a Journey Map](https://raw.githubusercontent.com/Codecademy/docs/main/media/journey-map.jpg)
```

File: content/uiux/journey-map/journey-map.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+---
+Title: 'Journey Maps in UX'
+Description: 'The visualization of the process that a person goes through in order to accomplish a goal with the intent of improving the overall user experience by providing a holistic view of the user's journey. '
```
Comment (ishg-153) [inline content/uiux/journey-map/journey-map.md:3]:
```suggestion
Description: 'The visualization of the process that a user goes through to accomplish a goal using the application.'
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@DevGat718 I have suggested a few changes. Please make those.

File: content/uiux/journey-map/journey-map.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+---
+Title: 'Journey Maps in UX'
```
[the remark this answers is not in the document]
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md:2]: done

File: content/uiux/journey-map/journey-map.md (hunk 7 above)
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md:20]: done

File: content/uiux/journey-map/journey-map.md (hunk 8 above)
Comment (DevGat718) [reply content/uiux/journey-map/journey-map.md:3]: done

**Comment by DevGat718** [comment]
but other than that I made all changes just need to push them through. @ishg-153

**Comment by ishg-153** [comment]
> but other than that I made all changes just need to push them through. @ishg-153 

I would suggest you do the following:
1. run the command `git reset HEAD --hard` this will make sure that your local repo aligns with the last commit you made (remember this will remove all changes that haven't been added to the staging area)
2. now `git fetch ux-journey-map` to ensure that your local repo is up to date with your remote fork.
3. finally, make the suggested changes - add the file, commit, and push

**Comment by ishg-153** [comment]
@DevGat718 If you wish to avoid this you can also directly commit the suggestions.
image

**Comment by DevGat718** [comment]
Screenshot 2024-02-28 at 15 59 57


same issue

**Comment by DevGat718** [comment]
been trying to trouble shoot, should I just clone it to make a copy?

**Comment by DevGat718** [comment]
I created a new pull request w/ the final changes on a new branch (ux-journeymap-v2)

**Pull request closed without merging**